src/screens/ENSAssignRecordsSheet.js

```diff
@@ -56,73 +63,110 @@ export default function ENSAssignRecordsSheet() {
     ],
   });
 
+  const avatarRadius = 35;
+
+  const formIsEmpty = Object.values(values).join('');
+
   return (
-    <Box background="body" flexGrow={1} paddingTop="30px">
-      <Rows>
-        <Row>
-          <Box as={ScrollView} flexGrow={1}>
-            <Inset horizontal="19px" top="4px">
-              <Stack space="30px">
-                <Stack alignHorizontal="center" space="15px">
-                  <Heading size="26px" weight="heavy">
-                    {ensName}
-                  </Heading>
-                  <Text color="action" size="16px" weight="heavy">
-                    Create your profile
-                  </Text>
-                </Stack>
-                <TextRecordsForm
-                  onBlurField={onBlurField}
-                  onChangeField={onChangeField}
-                  selectedFields={selectedFields}
-                  values={values}
-                />
-              </Stack>
-            </Inset>
-          </Box>
-          <Box bottom="0px" position="absolute" right="0px">
-            <Inset bottom="19px" right="19px">
-              <HideKeyboardButton />
-            </Inset>
-          </Box>
-        </Row>
-        <Row height="content">
-          <Box paddingBottom="30px" style={{ height: keyboardHeight }}>
-            <Shadow />
-            <Rows>
-              <Row>
-                <Inset space="19px">
-                  <SelectableAttributesButtons
-                    onAddField={onAddField}
-                    onRemoveField={onRemoveField}
-                    selectedFields={selectedFields}
-                  />
-                </Inset>
-              </Row>
-              <Row height="content">
-                <SheetActionButtonRow>
-                  <TintButton color="secondary60" onPress={handlePressBack}>
-                    􀆉 Back
-                  </TintButton>
-                  <SheetActionButton
-                    label="Review"
-                    onPress={handlePressContinue}
-                    size="big"
-                    weight="heavy"
+    <AccentColorProvider color={avatarColor}>
+      <Box background="body" flexGrow={1}>
+        <Rows>
+          <Row>
+            <Box as={ScrollView} flexGrow={1}>
+              <Box
+                background="accent"
+                height={{ custom: 125 }}
+                marginBottom={{ custom: 70 }}
+              >
+                <Cover alignHorizontal="center">
+                  <Box
+                    background="swap"
```

Comment:
I thought that the placeholder cover would be this, minus the "Add Cover" text:

<img width="449" alt="Screen Shot 2022-02-15 at 1 02 16 pm" src="https://user-images.githubusercontent.com/7336481/153978590-565e9abe-d8d8-4bfb-be3b-cf88d4d6f63a.png">

Wonder if we should do the same with avatar?